src/commands/publish/mod.rs

```diff
@@ -158,7 +157,10 @@ pub fn upload_buckets(
     let mut asset_manifest = None;
     for namespace in &target.kv_namespaces() {
         if let Some(bucket) = &namespace.bucket {
-            if bucket.is_empty() {
+            // We don't want folks setting their bucket to the top level directory,
+            // which is where wrangler commands are always called from.
+            let current_dir = env::current_dir()?;
+            if bucket.as_os_str() == current_dir {
```

Comment:
can we make issue for adding test for this (and maybe other workers sites stuff)

Reply:
Probably!

Reply:
https://github.com/cloudflare/wrangler/issues/858